src/lib/css/mixins/selection.scss

```diff
@@ -0,0 +1,185 @@
+// type can be: checkbox, radio or switch
+@mixin selection($type) {
+  width: auto;
+  height: auto;
+  line-height: normal;
+  white-space: nowrap;
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+
+  @if $type == checkbox or $type == radio {
+    > input {
+      width: 1.5rem;
+      height: 1.5rem;
+      opacity: 0;
+      background: red !important;
```

Comment:
Why this?

Reply:
Thanks, probably left over from debugging.